core/src/main/java/hudson/cli/Connection.java

```diff
@@ -113,6 +114,7 @@ public void writeObject(Object o) throws IOException {
     /**
      * Receives an object sent by {@link #writeObject(Object)}
      */
+    @SuppressFBWarnings(value = "OBJECT_DESERIALIZATION", justification = "Not used. We should just remove it.")
```

Comment:
If it Isn't used can we just remove it?

Reply:
I'm in favor of that, but there is generally a lot of resistance to removing any public API. The class has been deprecated for almost a year, so maybe we can remove it now. I would put that in a separate PR, though it could be reasonable to put it here.